app/Template/board/table_column.php

```diff
@@ -8,8 +8,8 @@
 
         <!-- column in collapsed mode -->
         <div class="board-column-collapsed">
-            <small class="board-column-header-task-count" title="<?= t('Show this column') ?>">
-                <span id="task-number-column-<?= $column['id'] ?>"><?= $column['nb_tasks'] ?></span>
+            <small class="board-column-header-task-count" title="<?= t('Task count') ?>">
+                <span id="task-number-column-<?= $column['id'] ?>"><span class="ui-helper-hidden-accessible"><?= t('Task count') ?>: </span><?= $column['nb_tasks'] ?></span>
```

Comment:
Same comment as this one: https://github.com/kanboard/kanboard/pull/4617#discussion_r499268766

Reply:
Colons now removed so we can simply re-use the existing translation terms